### PR TITLE
e2e test for the range-key option

### LINKED ISSUE #134: Create E2E test for range queries with `range-key`
We should add E2E test for the new `range-key` functionality

### LINKED ISSUE #145: Fix typo in the range-key docs
There is a typo in the docs about the range key, namely here:
```
query {
    findUserPurchasesInTime(id: "2", timestampFrom: 1, timestampTo: 4) {
    productId,
    price
    {
        total
    }
  }
}
```

 It should be `userId` and not `id`

### PR DESCRIPTION
**Pull request by DawidNiezgodka**
added an e2e test for range-key, updated the justfile. resolves #145 and resolves #134

### CODE REVIEW

File: e2e/functional/key-range/result-range.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[
```
Comment (torbsto) [inline e2e/functional/key-range/result-range.json:1]: Those aren't the expected results, right? The `productId` is the same for each and the totals aren't even in the input data.

Comment (DawidNiezgodka) [reply e2e/functional/key-range/result-range.json:1]: ah, ye, sry, I took the result from the docs but copied the one that relates to range-query and not range-query with key option

File: e2e/functional/key-range/result-range.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      {
+        "productId": 123,
+        "price": {
+          "total": 30
```
Comment (torbsto) [inline e2e/functional/key-range/result-range.json:5]: Does this work? I'd assume it must be `30.00`

Comment (DawidNiezgodka) [reply e2e/functional/key-range/result-range.json:5]: I believe I copied it directly from Altair and not wrote it by hand so it should function. I will check it later as I'm having some problems creating topics atm.

**Review by torbsto — APPROVED** [review]

**Pull request merged**